### PR TITLE
[SYTO-884] Style Video Modal

### PR DESCRIPTION
**Pull request by sebasrgl**
https://jira.natgeo.com/browse/SYTO-884

This ticket implement the Video Modal on mortar.
There is a refactor on the Modal Component to allow children and render them inside the modal content.

#### How to test
1. ``gulp serve``
2. Open /components
3. Scroll down until the Video Modal section
4. Click on "Launch Modal"
5. See the video with the proper information.

### CODE REVIEW

File: src/modules/modals/Modal.js [hunk 1]
```diff
@@ -13,14 +13,20 @@ class Modal extends Component {
   }
 
   componentWillMount() {
-    const targetIdNode = document.getElementById(this.props.targetId);
-    const modalContent = targetIdNode.innerHTML;
+    //This is a temporal fix when the in the modal don't render content in the
```
Comment (pmourelle) [inline src/modules/modals/Modal.js]: So, should this be removed? Or a new ticket created and referenced within this comment.

File: src/modules/modals/Modal.js [hunk 2]
```diff
@@ -13,14 +13,20 @@ class Modal extends Component {
...
+    //DOM.
+    try {
+      const targetIdNode = document.getElementById(this.props.targetId);
+      const modalContent = targetIdNode.innerHTML;
 
-    targetIdNode.parentNode.removeChild(targetIdNode);
+      targetIdNode.parentNode.removeChild(targetIdNode);
```
Comment (pmourelle) [inline src/modules/modals/Modal.js]: Are you removing the same node you're standing on? (fry)
That will also remove `modalContent` I guess.

Comment (pmourelle) [inline src/modules/modals/Modal.js]: After discussing with Sebas, this code will be tackled down in ticket SYTO-887.

File: src/modules/videomodal/_styles.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
+
+.mt3_video-modal {
+
+  &-container {
+    margin: auto;
+    max-width: 800px;
```
Comment (pmourelle) [inline src/modules/videomodal/_styles.scss]: Maybe there's already a breakpoint or variable that could be used in this case?

File: src/modules/videomodal/_styles.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      border-left: 1px solid $mt3_color-gray-40;
+    }
+  }
+
+  &__head {
+    margin: 10px 0 40px;
+  }
+
+  &__title {
+    font-family: $mt3_haas-stack;
```
Comment (pmourelle) [inline src/modules/videomodal/_styles.scss]: Isn't there any heading/class already defining styles like these?

Comment (sebasrgl) [inline src/modules/videomodal/_styles.scss]: Fixed!!!

File: src/modules/modals/Modal.js (hunk 1 above)
Comment (sebasrgl) [inline src/modules/modals/Modal.js]: This will be addressed on ticket SYTO-887

File: src/modules/videomodal/_styles.scss (hunk 3 above)
Comment (sebasrgl) [inline src/modules/videomodal/_styles.scss]: Added a variable in the scss file due to Mortar does n't handle a specific variable for that bp.

**Comment by pmourelle** [comment]
LGTM!

**Pull request merged**